scripts/gen-clients.sh

```diff
@@ -5,8 +5,8 @@ set -auexo pipefail
 # This script requres a checkout of https://github.com/kubernetes/code-generator in ../
 # For more information read https://blog.openshift.com/kubernetes-deep-dive-code-generation-customresources/
 
-echo -e "Cleanup previously generated code..."
-rm -rf pkg/client $(find ./pkg -name 'zz_*.go')
+# echo -e "Cleanup previously generated code..."
```

Comment:
do we want to uncomment this ?

Reply:
I would prefer to not use it, the script is not automated anywhere, and you will need to clone code-gen to make the script work